src/Model/PickupPoint.php

```diff
@@ -91,23 +92,29 @@ public function setCountry(string $country): void
         $this->country = $country;
     }
 
-    public function getLatitude(): ?string
+    public function getLatitude(): ?float
     {
         return $this->latitude;
     }
 
-    public function setLatitude(?string $latitude): void
+    public function setLatitude(?float $latitude): void
     {
+        Assert::nullOrGreaterThanEq($latitude, -90);
+        Assert::nullOrLessThanEq($latitude, 90);
```

Comment:
You could use `Assert::nullOrRange()`, I think.

Reply:
Didn't know about that one. Thanks :)